### PR TITLE
feat(messenger): use URLs for group invites

### PR DESCRIPTION
**Pull request by glouvigny**

### CODE REVIEW

File: api/bertymessenger.proto [hunk 1]
```diff
@@ -13,6 +13,9 @@ service MessengerService {
   // InstanceShareableBertyID returns a Berty ID that can be shared as a string, QR code or deep link.
   rpc InstanceShareableBertyID(InstanceShareableBertyID.Request) returns (InstanceShareableBertyID.Reply);
 
+  // InstanceShareableBertyGroup returns a Berty Group that can be shared as a string, QR code or deep link.
+  rpc InstanceShareableBertyGroup(InstanceShareableBertyGroup.Request) returns (InstanceShareableBertyGroup.Reply);
```
Comment (moul) [inline api/bertymessenger.proto:17]: Not sure about the naming here, the first one was about sharing the berty ID of the current instance

File: go/pkg/bertymessenger/api.go [hunk 2]
```diff
@@ -151,13 +140,130 @@ func (s *service) ParseDeepLink(ctx context.Context, req *ParseDeepLink_Request)
...
+	return &ret, nil
+}
+
+func (s *service) InstanceShareableBertyGroup(ctx context.Context, request *InstanceShareableBertyGroup_Request) (*InstanceShareableBertyGroup_Reply, error) {
+	grpInfo, err := s.protocol.GroupInfo(ctx, &bertytypes.GroupInfo_Request{
+		GroupPK:              request.GroupPK,
+		ContactPK:            nil,
+		XXX_NoUnkeyedLiteral: struct{}{},
+		XXX_unrecognized:     nil,
+		XXX_sizecache:        0,
```
Comment (moul) [inline go/pkg/bertymessenger/api.go:217]: can this be removed?

**Comment by moul** [comment]
> Should we deprecate `bertyprotocol.MultiMemberGroupInvitationCreate`?

To take this decision, we need to start maintaining a tested, simple hello world app in the monorepo, that uses bertyprotocol without bertymessenger

File: api/bertymessenger.proto (hunk 1 above)
Comment (glouvigny) [reply api/bertymessenger.proto:17]: Ok, renaming to `ShareableBertyGroup`

File: go/pkg/bertymessenger/api.go (hunk 2 above)
Comment (glouvigny) [reply go/pkg/bertymessenger/api.go:217]: My bad, thank you

**Comment by glouvigny** [comment]
> To take this decision, we need to start maintaining a tested, simple hello world app in the monorepo, that uses bertyprotocol without bertymessenger

Ok, no need to rush. I'll keep it as is for now.

**Pull request merged**